test/front-end.test.js

```diff
@@ -0,0 +1,83 @@
+const Application = require('spectron').Application;
+const assert = require('assert');
+
+describe('Front-end tests', function () {
```

Comment:
Please, use arrow function here

Reply:
Normally I would, but I won't be able to use `this.timeout()` if I use the arrow function here.

Reply:
You are right, we should use `function()` on the `describe` block on the other tests